src/products/components/CartActions/CartActions.tsx

```diff
@@ -1,15 +1,42 @@
-import React from "react";
+import React, { FC, useState } from "react";
 import { CartActionsStyled } from "./CartActions.styled";
 import { Button, useTranslate } from "../../../_shared";
 import { Quantity } from "../Quantity";
+import { useCartActions } from "../../../cart";
 
-export const CartActions = () => {
+interface Props {
+  id: number;
+}
+
+export const CartActions: FC<Props> = ({ id }) => {
   const translate = useTranslate();
+  const [qty, setQty] = useState(1);
+  const { addToCart } = useCartActions();
+
+  const handleCLickMinus = () => {
+    if (qty === 1) return;
+    setQty((prevState) => --prevState);
+  };
+
+  const handleClickPlus = () => {
+    setQty((prevState) => ++prevState);
+  };
 
   return (
     <CartActionsStyled>
-      <Quantity />
-      <Button variant="orange">{translate("productAction.add")}</Button>
+      <Quantity
+        qty={qty}
+        handleCLickMinus={handleCLickMinus}
+        handleClickPlus={handleClickPlus}
+      />
+      <Button
+        variant="orange"
+        onClick={() => {
```

Comment:
conceptually orange is better to be a color instead of a variant, if we want to strict consider the design system